server/models/Organization.ts

```diff
@@ -26,7 +26,7 @@ function getByFilter(
 function getOrganizationLinks(organization) {
   const links = {
     featured_trees: `/trees?organization_id=${organization.id}&limit=20&offset=0`,
-    associated_planters: `/planters?organization_id=${organization.id}&limit=20&offset=0`,
+    associated_planters: `/planters?organization_id=${organization.id}&limit=20&offset=0&order_by=about`,
```

Comment:
@MundhadHarsh did you test this feature, I mean, with this order_by parameter, the SQL will try to add corresponding phrase?

Reply:
No, I have not tested the feature. I made changes given in the description of #243